<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilmConstInfoClass</class>
 <widget class="QDialog" name="FilmConstInfoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>394</width>
    <height>379</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FilmConstInfo</string>
  </property>
  <widget class="QLabel" name="lb_FilmNameInfo">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>151</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ink Free</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Seria Movie Info</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_prodYear">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>65</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Production Year:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_name">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>35</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_genre">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>125</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Genre:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pB_ok">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>340</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_prodCountry">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>95</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Production Country:</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stWidg_specInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>291</width>
     <height>121</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="pg_sinMovSpInfo">
    <widget class="QLabel" name="lb_minsAmount">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>35</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Minutes amount:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_hoursAmount">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Hours amount:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_setHoursAmount">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>5</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>*hours</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_setMinsAmount">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>35</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>*minutes</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pg_serialSpInfo">
    <widget class="QLabel" name="lb_epsAmount">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Episodes Amount:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_seasonNum">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>131</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Season Number:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_epLen">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Episode Lenght:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_setSeasNum">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>*seasonNum</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_setEpsNum">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>40</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>*epsAmount</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_setEpLen">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>*epLenght</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="pg_serMovspInfo">
    <widget class="QLabel" name="lb_hoursAmount_serM">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>5</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Hours Amount:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_minsAmount_serM">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>35</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Minutes Amount:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_status">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>95</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Status:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_seriaNum_serM">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>65</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>Seria Number:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_setHoursAmount_serM">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>5</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>*hours</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_setMinsAmount_serM">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>35</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>*minutes</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_setSeriaNum">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>65</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>*seria num</string>
     </property>
    </widget>
    <widget class="QLabel" name="lb_setStatus">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>95</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>*status</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="lb_setName">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>35</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>*name</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_setProdYear">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>65</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>*year</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_setProdCountry">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>95</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>*country</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_setGenre">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>125</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>*genre</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_setRating">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>275</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>*rating</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_rating">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>275</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>Rating:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_ID">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_setID">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Comic Sans MS</family>
     <pointsize>12</pointsize>
     <italic>true</italic>
    </font>
   </property>
   <property name="text">
    <string>*id</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pB_ok</sender>
   <signal>clicked()</signal>
   <receiver>FilmConstInfoClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>195</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
